packages/grant-explorer/src/features/round/ViewRoundPage.tsx

```diff
@@ -259,65 +259,72 @@ function AfterRoundStart(props: {
-            style={{ borderRadius: "20px" }}
-          >
-            {round.payoutStrategy?.strategyName &&
-              getRoundType(round.payoutStrategy?.strategyName)}
-          </p>
-          <div className="flex text-grey-400 mb-1">
-            <p className="mr-4 text-sm">
-              <span className="mr-1">Round starts on:</span>
-              <span className="mr-1">
-                {formatUTCDateAsISOString(round.roundStartTime)}
-              </span>
-              <span>{getUTCTime(round.roundStartTime)}</span>
-            </p>
-            <p className="text-sm">
-              <span className="mr-1">Round ends on:</span>
-
-              {!isInfiniteDate(round.roundEndTime) && (
-                <>
+          <div className="flex flex-col md:items-center md:justify-between md:gap-8 md:flex-row md:mb-0 mb-4">
+            <div>
+              <p data-testid="round-title" className="text-3xl mb-5">
+                {round.roundMetadata?.name}
+              </p>
+              <p
+                data-testid="round-badge"
+                className="text-sm text-gray-900 h-[20px] inline-flex flex-col justify-center bg-grey-100 px-3 mb-4"
+                style={{ borderRadius: "20px" }}
+              >
+                {round.payoutStrategy?.strategyName &&
+                  getRoundType(round.payoutStrategy?.strategyName)}
+              </p>
+              <div className="flex text-grey-400 mb-1">
+                <p className="mr-4 text-sm">
+                  <span className="mr-1">Round starts on:</span>
                   <span className="mr-1">
-                    {formatUTCDateAsISOString(round.roundEndTime)}
+                    {formatUTCDateAsISOString(round.roundStartTime)}
                   </span>
+                  <span>{getUTCTime(round.roundStartTime)}</span>
+                </p>
+                <p className="text-sm">
+                  <span className="mr-1">Round ends on:</span>
+
+                  {!isInfiniteDate(round.roundEndTime) && (
+                    <>
+                      <span className="mr-1">
+                        {formatUTCDateAsISOString(round.roundEndTime)}
+                      </span>
+
+                      <span>{getUTCTime(round.roundEndTime)}</span>
+                    </>
+                  )}
+                  {isInfiniteDate(round.roundEndTime) && (
+                    <>
+                      <span>No End Date</span>
+                    </>
+                  )}
+                </p>
+              </div>
+
+              <p className="text-grey-400 text-sm flex gap-2 mb-4">
+                <span>Deployed on:</span>
+                <div className="flex">
+                  <img
+                    className="w-4 h-4 mt-0.5 mr-1"
+                    src={CHAINS[chainId].logo}
+                    alt="Round Chain Logo"
+                  />
+                  <span>{CHAINS[chainId].name}</span>
```

Comment:
Dunno if it could happen but if `CHAINS[chainId]` is undefined, it will throw an uncaught error.
Might be a good idea to use optional chaining here just to be safe `CHAINS[chainId]?.logo`.

Reply:
updated